cli/mobycli/exec.go

```diff
@@ -111,6 +111,18 @@ func Exec(_ *cobra.Command) {
 	}
 	commandArgs := os.Args[1:]
 	command := metrics.GetCommand(commandArgs)
+	if (command == "build" || command == "pull") && !metrics.HasQuietFlag(commandArgs) {
+		var image string
+		if command == "pull" {
+			for _, a := range commandArgs[1:] {
+				if !strings.HasPrefix(a, "--") {
+					image = a
```

Comment:
This probably won't work for `docker build` as it takes the build context as positional argument, not an image.

Also note that;

- build may not build an image (in case `--output` is used)
- build allows controlling the output format (plain/tty), ik which case we may have to adjust the output accordingly

Reply:
Yes, this part is only for pull, not build. In case of a build we don't care that much as we already have a shortcut to get the most recently built image.

But I'll check:
- the output so it's only displayed when we build an image
- I'll check the output format

Reply:
The hint already handle plain/tty, I'll just check if there's other way to enable/disable it for build for instance
<img width="672" alt="image" src="https://github.com/docker/compose-cli/assets/1542321/e2fb0f16-0bda-43f4-9b35-9e5216bb0823">